<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEntryDialog</class>
 <widget class="QDialog" name="NewEntryDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Entry</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.ico</normaloff>:/images/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="FirstNameLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="FirstNameEntry"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="LastNameLabel">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="LastNameEntry"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="stAddressText">
        <property name="text">
         <string>St. Address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="stAddresField"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ZipCodeLabel">
        <property name="text">
         <string>Postal Code</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="ZipCodeEntry"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="AddContactOptions">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AddContactOptions</sender>
   <signal>accepted()</signal>
   <receiver>NewEntryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddContactOptions</sender>
   <signal>rejected()</signal>
   <receiver>NewEntryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
